package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "phaser3-nineslice",
   "description": "A 9-slice implementation for Phaser 3",
-  "version": "0.2.2",
-  "main": "dist/nineslice.js",
+  "version": "0.2.3",
```

Comment:
well shit. It looks i forgot to push v0.3.0 to master; I'll need to do that and merge your change after any conflict resolution